manual/src/threelang/stellar/load.md

```diff
@@ -0,0 +1,17 @@
+# Load Action
+
+> Load a client, connecting to the rpc endpoint at the given network and loading a keypair from the given secret.
```

Comment:
For loading I don't think it should be an action. All keys should go into one action at the top of the md file. That will make it easier for the user. Check with Omar, I added a comment about this on his PR.

Reply:
i agree, will remove from all other modules also